rasa/core/policies/ted_policy.py

```diff
@@ -320,6 +318,13 @@ def train(
     ) -> None:
         """Train the policy on given training trackers."""
 
+        if not training_trackers:
+            logger.error(
```

Comment:
it's rather a warning than an error

Reply:
it is an error below if model data is empty

Reply:
I just reused the old code